modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1786 @@
+======
+
+.Create a constraint requiring `Actor` nodes to have a unique combination of `firstname` and `surname` properties as a node key
+[source, cypher]
+----
+CREATE CONSTRAINT actor_fullname
+FOR (actor:Actor) REQUIRE (actor.firstname, actor.surname) IS NODE KEY
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+.Create a composite relationship key constraint label on multiple properties label:new[Introduced in 5.7]
+======
+
+.Create a constraint requiring `KNOWS` relationships to have a unique combination of `since` and `how` properties as a relationship key
+[source, cypher]
+----
+CREATE CONSTRAINT knows_since_how
+FOR ()-[knows:KNOWS]-() REQUIRE (knows.since, knows.how) IS RELATIONSHIP KEY
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[create-key-constraint-compliant-data]]
+==== Create data that complies with existing key constraints
+
+.Create a node that complies with existing node key constraints
+======
+
+.Create an `Actor` node with `firstname` and `surname` properties that are not already in the database
+[source, cypher]
+----
+CREATE (actor:Actor {firstname: 'Keanu', surname: 'Reeves'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 label, created 1 node, set 2 properties.
+----
+
+======
+
+
+.Create a relationship that complies with existing relationship key constraints
+======
+
+.Create a `KNOWS` relationship with `since` and `how` properties that are not already in the database
+[source, cypher]
+----
+CREATE (:Actor {firstname: 'Jensen', surname: 'Ackles'})-[:KNOWS {since: 2008, how: 'coworkers', friend: true}]->(:Actor {firstname: 'Misha', surname: 'Collins'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 2 labels, created 2 nodes, set 7 properties, created 1 relationship.
+----
+
+======
+
+
+[role=label--new-5.16]
+[[create-constraint-with-parameter]]
+=== Create a constraint with a parameter
+
+All constraint types can be created with a parameterized name.
+
+.Create a node property uniqueness constraint using a parameter
+======
+
+.Parameters
+[source, parameters]
+----
+{
+  "name": "node_uniqueness_param"
+}
+----
+
+.Create a node property uniqueness constraint with a parameterized name
+[source, cypher]
+----
+CREATE CONSTRAINT $name
+FOR (book:Book) REQUIRE book.prop1 IS UNIQUE
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+.Create a relationship property existence constraint using a parameter
+======
+
+.Parameters
+[source, parameters]
+----
+{
+  "name": "rel_exist_param"
+}
+----
+
+.Create a relationship property existence constraint with a parameterized name
+[source, cypher]
+----
+CREATE CONSTRAINT $name
+FOR ()-[wrote:WROTE]-() REQUIRE wrote.published IS NOT NULL
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[multiple-constraints-on-same-property-combinations]]
+=== Multiple constraints on the same property combinations
```

Comment:
it feels a bit odd to have this, and then a bit later the `Handling existing constraints when creating a constraint` and then later again the `Creating an already existing constraint will fail`. It feels like all of these bits are related to each other and are now very split apart 🤔 

It might also be nice in this info section to mention which constraints can't exist on the same property combinations, like key and uniqueness or differing property type ones. Instead of just listing 'happy' examples

Reply:
fixed in my last commit